assemblers/org.osbuild.tar

```diff
@@ -7,13 +7,19 @@ import sys
 
 def main(tree, output_dir, options):
     filename = options["filename"]
-    compression = options["compression"]
+    compression = options.get("compression", None)
 
-    if compression not in {"bzip2", "xz", "lzip", "lzma", "lzop", "gzip"}:
+    if compression not in {"bzip2", "xz", "lzip", "lzma", "lzop", "gzip", None}:
```

Comment:
What about moving this part below the condition `if compression is not None:` so we don't have the `None` here?